apps/holidays-viewer/templates/add.html

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.html" %}
+{% block content %}
+  <div class="instructions m-3">
+    <p>Add a new test holiday
+  </div>
+  <div class="base rounded shadow bg-white m-3">
+    <form class="m-auto" action="" method="POST">
+      <div class="form-group">
+        <div class="form-field">
+          <div class="label-field">Date( [Month Day] e.g. January 1 )</div>
```

Comment:
Change: "Date( [Month Day] e.g. January 1 )" to "Date( [MMMM dd] e.g. January 1 )"